fetch.bs

```diff
@@ -8316,6 +8169,14 @@ resource — for non-<a>CORS requests</a> as well as <a>CORS
 requests</a> — and do not use `<code>Vary</code>`.
 
 
+<h3 class=no-num id=websocket-protocol oldids=websocket-connections,websocket-opening-handshake,fail-the-websocket-connection,the-websocket-connection-is-established>WebSockets</h2>
+
+<p class=note>Fetch used to define
```

Comment:
A section with only a note looks strange. Maybe just make this a normal paragraph.